packages/multi-test/src/app.rs

```diff
@@ -492,9 +489,9 @@ mod test {
         assert_eq!(funds, coins(5, "eth"));
 
         // reflect count updated
-        let qres: PayoutCountResponse = app
+        let qres: payout::CountResponse = app
```

Comment:
All of these type names get better

Reply:
My hint: if you have 3+ names in your file/module with exactly the same prefix, you want to have a submodule ;)